client/src/components/landing/components/CampersImage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Media from 'react-responsive';
+import { Spacer, ImageLoader } from '../../helpers';
+import wideImg from '../../../assets/images/landing/wide-image.png';
+
+const propTypes = {
+  page: PropTypes.string
+};
+
+const LARGE_SCREEN_SIZE = 1200;
+
+const ImageConfig = {
+  donate: {
+    spacerSize: 0,
+    height: 345,
+    width: 585
+  },
+  landing: {
+    spacerSize: 2,
+    height: 442,
+    width: 750
+  }
+};
+
+function CampersImage({ page }) {
+  const { spacerSize, height, width } = ImageConfig[page];
+  return (
+    <Media minWidth={LARGE_SCREEN_SIZE}>
+      <Spacer size={spacerSize} />
+      <ImageLoader
+        alt='Freecodecamp students at a local study'
```

Comment:
```suggestion
        alt='freeCodeCamp students at a local study group in South Korea.'
```
It's a bit weird since the caption is below, but I think this is better.